src/main/java/com/sweep/projectsweep/services/AvailabilityService.java

```diff
@@ -0,0 +1,22 @@
+package com.sweep.projectsweep.services;
+
+import com.sweep.projectsweep.jooq.tables.pojos.Availability;
+import com.sweep.projectsweep.jooq.tables.pojos.Profile;
+import com.sweep.projectsweep.repositories.AvailabilityRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class AvailabilityService {
+    @Autowired
+    AvailabilityRepo availabilityRepo;
```

Comment:
Side note, we gotta fix the formatter to add some new lines here. No need to do that in this PR though.